bin/qunit-testrunner.js

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env node
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+const path = require('path');
+const yargs = require('yargs');
+const getPort = require('get-port');
+const serveStatic = require('serve-static');
+const serveIndex = require('serve-index');
+const bodyParser = require('body-parser');
+const TestRunner = require('../src/testrunner');
+const WebServer = require('../src/webserver');
```

Comment:
Since those 2 modules export now function the variable name shouldn't be capitalized